build-app.sh

```diff
@@ -1,3 +1,2 @@
 #!/bin/bash
-
-docker build -t hoeghh/age:1.0 ./app
+docker build -t "$usern/age:1.0" ./app
```

Comment:
The 'usern' here might be more appropriately named 'registry' or 'repository'. Its normally not related to a 'user' per se.